operators/nvidia-network-operator/1.2.1/manifests/nvidia-network-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,500 @@
+                imagePullPolicy: IfNotPresent
+                livenessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 8081
+                  initialDelaySeconds: 15
+                  periodSeconds: 20
+                name: manager
+                readinessProbe:
+                  httpGet:
+                    path: /readyz
+                    port: 8081
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
+                resources:
+                  limits:
+                    cpu: 300m
+                    memory: 100Mi
+                  requests:
+                    cpu: 200m
+                    memory: 50Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsUser: 65532
+              serviceAccountName: nvidia-network-operator-controller-manager
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: nvidia-network-operator-controller-manager
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - sriov
+  - networking
+  links:
+  - name: Network Operator Documentation
+    url: https://docs.nvidia.com/networking/display/COKAN10/Network+Operator
+  - name: GitHub repository
+    url: https://github.com/Mellanox/network-operator/
+  maintainers:
+  - email: nvidia-network-operator-support@nvidia.com
+    name: NVIDIA
+  maturity: alpha
+  provider:
+    name: NVIDIA
+    url: https://github.com/Mellanox/network-operator/
+  relatedImages:
+  - image: nvcr.io/nvidia/mellanox/mofed-5.6-1.0.3.3@sha256:0a5108443c64fc013984be500e7db3d89c04418446f0bbc3241ed6e1c449b773
+    name: mofed
+  - image: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin@sha256:16a53286fecdb1e587d3c4c042078974674c3e86c9e98d7dae282f6eb4ee2d8c
+    name: sriov-network-device-plugin
+  - image: nvcr.io/nvidia/cloud-native/k8s-rdma-shared-dev-plugin@sha256:941ad9ff5013e9e7ad5abeb0ea9f79d45379cfae88a628d923f87d2259bdd132
+    name: rdma-shared-device-plugin
+  version: 1.2.1
```

Comment:
Do we need a `replaces` entry?

`replaces: nvidia-network-operator.v1.2.0`

I see it in GPU Operator for example:
https://github.com/redhat-openshift-ecosystem/certified-operators/blob/main/operators/gpu-operator-certified/v1.10.1/manifests/gpu-operator-certified.clusterserviceversion.yaml#L684

Reply:
done